packages/@aws-cdk/aws-eks/lib/kubectl-provider.ts

```diff
@@ -16,7 +47,7 @@ export class KubectlProvider extends NestedStack {
    */
   public readonly role: iam.IRole;
 
-  public constructor(scope: Construct, id: string) {
+  public constructor(scope: Construct, id: string, props: KubectlProviderProps) {
```

Comment:
All fields are optional:

```suggestion
  public constructor(scope: Construct, id: string, props: KubectlProviderProps = { }) {
```

Reply:
👍